src/js/webrequest.js

```diff
@@ -1162,11 +1162,12 @@ function dispatcher(request, sender, sendResponse) {
   // https://bugzilla.mozilla.org/show_bug.cgi?id=1787379
   // https://github.com/uBlockOrigin/uBlock-issues/issues/1992#issuecomment-1058056302
   //
-  // TODO remove all sender.origin fallbacks once minimum supported versions
-  // TODO equal or exceed 80 (Chromium) and 126 (Firefox) in all builds
+  // TODO remove sender.origin sender.url/request.frameUrl fallbacks
+  // TODO once minimum supported versions equal or exceed
+  // TODO 80 (Chromium) and 126 (Firefox) in all builds
   if (utils.hasOwn(sender, "origin") ?
-    sender.origin + '/' !== chrome.runtime.getURL('') :
-    !sender.url.startsWith(chrome.runtime.getURL(''))) {
+    sender.origin == "null" || sender.origin + '/' !== chrome.runtime.getURL('') :
```

Comment:
"null" feels like a bug that browsers will hopefully fix. should we also check for sender.origin being null?

Reply:
I think `"null"` is intentional, the [`initiator` property](https://github.com/EFForg/privacybadger/blob/4bf849b0f120641f6215a1b2afa3afa3db86cd38/src/lib/webrequestUtils.js#L33) of webRequest callback details objects can be `"null"` too.

>The origin where the request was initiated. This does not change through redirects. If this is an opaque origin, the string 'null' will be used.

-- https://developer.chrome.com/docs/extensions/reference/api/webRequest

Reply:
I'm not aware of any cases where `MessageSender.origin` is falsy.